### PR TITLE
Add utility pallet tests

### PR DESCRIPTION
**Pull request by niazhussain**

### CODE REVIEW

**Comment by sayyed-kamran** [comment]
Added functions for integration testing of utility pallet to the standalone chain

File: test.py [hunk 1]
```diff
@@ -6,16 +6,22 @@
...
+from tools.pallet_utility_test import pallet_utility_batch_all_test
 # from tools.pallet_asset_evm_with_rpc import pallet_asset_evm_test
 
 
 if __name__ == '__main__':
     # pallet_batchall_test()
-    pallet_multisig_test()
-    pallet_transaction_test()
-    pallet_did_test()
+    #pallet_multisig_test()
```
Comment (sfffaaa) [inline test.py:16]: Please uncomment the below lines.

File: tools/pallet_utility_test.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
+import sys
```
Comment (sfffaaa) [inline tools/pallet_utility_test.py:1]: Please fix all `flake8`'s errors and warnings.

File: tools/pallet_utility_test.py [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    )
+
+#source account
+kp_src = Keypair.create_from_uri('//Alice')
+#destination account
+kp_dst=Keypair.create_from_uri('//Charlie')
+
+#One vaid and one invalid transaciton will added to pallet::utility::batch for execution
+# The restul of valid transaction will not be reverted due to an invalid transaton int batch      
+def pallet_utility_batch_test():
```
Comment (sfffaaa) [inline tools/pallet_utility_test.py:21]: Please change the function naming, add a successful path and change the code structure.

1. The function name should give users a hit on what that is.
```
pallet_utility_batch_test maybe can become batch_not_revert
pallet_utility_batch_all_test can become batch_all_revert
```

2. Please add two more test cases for the successful path.
  1. Two calls in the batch executed successfully
  2. Two calls in the batch all executed successfully. 

3. When you add the new test cases, I think we have to change the code structure, which means we may need one pallet_utility_batch_test, and when we run it, it'll test all related tests (batch but not revert + batch successfully). The same as batch_all.

File: test.py (hunk 1 above)
Comment (sayyed-kamran) [reply test.py:16]: Uncommented all tests EXCEPT the ones that were already commented

File: tools/pallet_utility_test.py (hunk 2 above)
Comment (sayyed-kamran) [reply tools/pallet_utility_test.py]: Fixed all flake8's  error and warnings EXCEPT the one:

"tools/pallet_utility_test.py:28:5: F841 local variable 'bal_src_before' is assigned to but never used"

Actually I am using this variable to get/display account balance of source account using: 
bal_src_after = show_account(substrate,kp_src.ss58_address, "Src bal after trans")

File: tools/pallet_utility_test.py (hunk 3 above)
Comment (sayyed-kamran) [reply tools/pallet_utility_test.py]: I changed  the structure of the file. created self explanatory  four test functions:

def pallet_utility_test():  # Main function that will cal all the test for pallet_utility
    all_valid_extrinsics_bath() # A valid transaction added twice to the batch and submitted via pallet::utility::batch.
    all_valid_extrinsics_bath_all() # A valid transaction added twice to the batch and submitted via pallet::utility::batch_all    
    atleast_one_invalid_extrinsic_bath() # One valid and one invalid transaction submitted via pallet::utility::batch. Valid 
                                                               # transaction will not be reverted due to invalid transaction
    atleast_one_invalid_extrinsic_bath_all() # One valid and one invalid transaction submitted via pallet::utility::batch_all.
                                                                    # Valid transaction will be reverted due to invalid transaction

**Review by sfffaaa — APPROVED** [review]

**Pull request merged**